index.js

```diff
@@ -0,0 +1,58 @@
+import Book from './modules/book.js';
+import Store from './modules/store.js';
+import ShowBooks from './modules/showBooks.js';
```

Comment:
* Kindly create all these modules you are importing as currently they have not been created and as a result most of the functionalities are missing 
![image](https://github.com/AlyAtalla/Awesome-books/assets/52497246/e1f45e11-4cb1-4bea-b597-3bad72e98fba)